### PR TITLE
Add vehicule planning and remove old calendar

### PR DESCRIPTION
**Pull request by Rodrive**

### CODE REVIEW

**Comment by Rodrive** [comment]
[Revoir le planning véhicules](https://trello.com/c/OU1m91xc/176-revoir-le-planning-v%C3%A9hicules)

File: components/planning/cars/calendar.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  <div class="calendar column">
+    <client-only>
+      <vue-cal
+        default-view="week"
+        locale="fr"
+        hide-view-selector
+        watch-real-time
+        dom-cells
+        :hide-weekdays="hideWeekDays"
+        :time-from="START_DAY_HOUR * 60"
```
Comment (Nainterceptor) [inline components/planning/cars/calendar.vue]: wasn't made configurable in a previous PR ?

File: components/planning/cars/calendar.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          .filter((d) => !this.$store.state.context.campus.workedDays.includes(d))
+        : [6, 7];
+    },
+    START_DAY_HOUR() {
+      return (this.$store.state.context.campus && this.$store.state.context.campus.workedHours)
+        ? this.$store.state.context.campus.workedHours.start : START_DAY_HOUR;
+    },
+    END_DAY_HOUR() {
+      return (this.$store.state.context.campus && this.$store.state.context.campus.workedHours)
+        ? this.$store.state.context.campus.workedHours.end : END_DAY_HOUR;
```
Comment (Nainterceptor) [inline components/planning/cars/calendar.vue]: Arg, dont bind the state directly, please use getters, then mapGetters

File: components/planning/cars/calendar.vue (hunk 1 above)
Comment (Nainterceptor) [reply components/planning/cars/calendar.vue]: Ok, it is, but it's not a const, maybe camelCase it ?

**Pull request merged**